src/components/List/ListItem.tsx

```diff
@@ -11,7 +11,7 @@ export type ListItemProps = {
   /** @default normal */
   rowSize?: ListRowSize;
 };
-const ListItem: React.FCC<ListItemProps> = (props) => (
+const ListItem: React.FC<React.PropsWithChildren<ListItemProps>> = (props) => (
```

Comment:
this could probably be FCC

Reply:
resolved here https://github.com/Orfium/orfium-ictinus/pull/720/commits/f50892c7c98ff1d524fd9d7c6393c46fc9782161